.github/workflows/performance-testing.yaml

```diff
@@ -75,16 +68,13 @@ jobs:
           chmod +x nb5
           ./nb5 --version
 
-      - name: Build JSON API Docker image
-        run: |
-          ./mvnw clean package -DskipTests ${{ matrix.build-ops }} 
-
       # Use docker-compose to start required stack
       # Print version of images used for verification
       - name: Start Docker Compose
         run: |
           cd docker-compose
-          ./start_dse_68_dev_mode.sh ${{ matrix.docker-flags }}
+          JSONTAG=$(git show -s --format="%H" main)
```

Comment:
allows running this workflow against PR branches - will still reference latest commit SHA on main branch.